msal/src/Microsoft.Identity.Client/Features/ConfidentialClient/ClientCredential.cs

```diff
@@ -45,8 +45,11 @@ public ClientCredential(ClientAssertionCertificate certificate)
         }
 
         internal ClientAssertionCertificate Certificate { get; private set; }
-        internal string Assertion { get; set; }
+        internal string Assertion { get;
+            set; }
         internal long ValidTo { get; set; }
```

Comment:
>ValidTo [](start = 22, length = 7)

Given this is completely internal now... however we might want to keep the expired functionality which was there before.

Reply:
#Resolved